app/controllers/stories/feeds_controller.rb

```diff
@@ -65,8 +65,8 @@ def signed_in_base_feed
     end
 
     def signed_out_base_feed
-      strategy = AbExperiment.get(experiment: AbExperiment::CURRENT_FEED_STRATEGY_EXPERIMENT, controller: self, user: current_user,
-                                  default_value: AbExperiment::ORIGINAL_VARIANT)
+      strategy = AbExperiment.get(experiment: AbExperiment::CURRENT_FEED_STRATEGY_EXPERIMENT, controller: self,
+                                  user: current_user, default_value: AbExperiment::ORIGINAL_VARIANT)
```

Comment:
I fixed this Rubocop line length error since it was preventing me from committing and pushing my changes.